docs/source/layout_guide.rst

```diff
@@ -63,8 +63,7 @@ being relative to the top left we anchor to, say the bottom right. That would lo
 .. code-block:: python
    :linenos:
 
-    button_layout_rect = pygame.Rect(0, 0, 100, 20)
-    button_layout_rect.bottomright = (-30, -20)
+    button_layout_rect = pygame.Rect(30, 20, 100, 20)
```

Comment:
Why are you changing the documentation if, as your PR says, you are changing the code to fit the documentation?

Reply:
The incident happened when I tried to use anchors without setting up a container.

```python
import pygame
import pygame_gui
from pygame._sdl2 import Window


if __name__=="__main__":
    # define
    fps = 60
    window_size = (1280, 720)

    # init
    pygame.init()
    
    # create window
    win = Window("pygm_player", window_size, resizable=True)
    win_surface = win.get_surface()
    ui_manager = pygame_gui.UIManager(window_size)

    # test gui

    button_layout_rect = pygame.Rect(0, 0, 100, 20)
    button_layout_rect.bottomright = (-30, -20)

    cancel_button = pygame_gui.elements.UIButton(relative_rect=button_layout_rect,
                                      text='pygame-gui.Cancel',
                                      manager=ui_manager,
                                      object_id='#cancel_button',
                                      anchors={
                                               'right': 'right',
                                               'bottom': 'bottom',
                                               })
    
    ok_button = pygame_gui.elements.UIButton(relative_rect=pygame.Rect(0, 0, 100, 20),
                                  text='pygame-gui.OK',
                                  manager=ui_manager,
                                  object_id='#ok_button',
                                  anchors={
                                           'right': 'right',
                                           })


    # eventloop
    running = True
    clock = pygame.time.Clock()
    while running:
        time_delta = clock.tick(fps)/1000.0
        # event pass
        for event in pygame.event.get():
            if event.type == pygame.QUIT:
                running = False
            
            ui_manager.process_events(event)

        # draw window
        ui_manager.update(time_delta)
        win_surface.fill(pygame.Color('#333333'))
        ui_manager.draw_ui(win_surface)
        
        # render
        win.flip()

    pygame.quit()
```
When I ran the script above, I only saw one button in the window. 

It behaves like a collection of bugs.

As you can see, things got worse when I tried to modify it.

Over the past two days, I have felt completely exhausted.

Okey，

I am the BUG

This is the first time I use pull requests, and I’m not sure what to do next.

I have reverted to the original version.

- Please close or remove the request. 

After all, to avoid using anchors without a defined container, perhaps I need to add a **`highlighted warning`** on the [Layout Guide](https://pygame-gui.readthedocs.io/en/latest/layout_guide.html) page.